Eatery/Controllers/Eateries/CollegetownEateriesMenuViewController.swift

```diff
@@ -0,0 +1,389 @@
+        super.init(nibName: nil, bundle: nil)
+        
+        mapView.delegate = self
+        
+        locationManager = CLLocationManager()
+        locationManager.delegate = self
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        
+        if CLLocationManager.locationServicesEnabled() {
+            switch (CLLocationManager.authorizationStatus()) {
+            case .authorizedWhenInUse:
+                locationManager.startUpdatingLocation()
+                mapView.showsUserLocation = true
+            case .notDetermined:
+                if locationManager.responds(to: #selector(CLLocationManager.requestWhenInUseAuthorization)) {
+                    locationManager.requestWhenInUseAuthorization()
+                }
+            default: break
+            }
+        }
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.navigationBar.isHidden = true
+        if #available(iOS 11.0, *) {
+            navigationItem.largeTitleDisplayMode = .never
+        }
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        navigationController?.navigationBar.isHidden = false
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        class CustomScrollView: UIScrollView{
+            override func touchesShouldCancel(in view: UIView) -> Bool {
+                if(view is UIButton){
+                    return false
+                }
+                return true
+            }
+        }
+        
+        outerScrollView = CustomScrollView()
+        outerScrollView.backgroundColor = .wash
+        outerScrollView.delegate = self
+        outerScrollView.showsVerticalScrollIndicator = false
+        outerScrollView.showsHorizontalScrollIndicator = false
+        outerScrollView.alwaysBounceVertical = true
+        outerScrollView.delaysContentTouches = false
+        view.addSubview(outerScrollView)
+        
+        ctownMenuHeaderView = CollegetownMenuHeaderView()
+        ctownMenuHeaderView.set(eatery: eatery, userLocation: userLocation)
+        ctownMenuHeaderView.isUserInteractionEnabled = true
+        outerScrollView.addSubview(ctownMenuHeaderView)
+        
+        backButton = UIButton()
+        backButton.setImage(UIImage(named: "back"), for: .normal)
+        backButton.isUserInteractionEnabled = true
+        backButton.addTarget(self, action: #selector(goBack), for: .touchUpInside)
+        outerScrollView.addSubview(backButton)
+        
+        informativeViews = [UIView]()
+        
+        for i in 0...2 {
+            let informativeView = UIView()
+            informativeView.backgroundColor = .white
+            informativeView.isUserInteractionEnabled = true
+            informativeViews.append(informativeView)
+            
+            var gesture : UITapGestureRecognizer!
```

Comment:
`let gesture: UITapGestureRecognizer`; no need for an implicitly unwrapped optional